crates/isograph_schema/src/magic_mutation_fields.rs

```diff
@@ -496,3 +353,94 @@ fn skip_arguments_contained_in_field_map(
         processed_field_map_items,
     ))
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
I am not sure I can come up with a meaningful type. Do you mean like any struct? I think it's important to match the directive structure, since we want to deny all other fields